### ISSUE TITLE
Implement MLP class and methods

### ISSUE THREAD

**Issue by ben-jamming** [ben-jamming/COMP-551#7]
Description: A Python class for the MLP model. The constructor should accept the activation function, number of hidden layers, and number of units in each hidden layer. Initialize weights, biases, and all MLP properties.
- [ ] Implement the `fit` function for the MLP class, taking training data (X and y) and hyperparameters (learning rate, gradient descent iterations) to train the model.
- [ ] Implement the `predict` function that takes a set of points X and outputs y_hat predictions.
- [ ] Implement an `evaluate_acc` method that inputs true labels y and target labels y_hat to output an accuracy score.
